pkgs/by-name/ol/ollama/package.nix

```diff
@@ -191,20 +187,13 @@ goBuild {
     make -j $NIX_BUILD_CORES
```

Comment:
this needs to be changed to 

`make dist -j $NIX_BUILD_CORES`

This fixes cuda for me.